app/src/pages/playground/fetchPlaygroundPrompt.ts

```diff
@@ -165,19 +178,53 @@ export const promptVersionToInstance = ({
       messages:
         "messages" in promptVersion.template
           ? promptVersion.template.messages.map((m) => {
-              const maybeTextPart = m.content.find(isTextPart);
-              if (isTextPart(maybeTextPart)) {
+              // select all parts
+              const textContent = (
+                m.content.map(asTextPart).filter(Boolean) as TextPart[]
+              )
+                .map((part) => part.text.text)
+                .join("");
```

Comment:
this is kinda a hack to combine no?

Reply:
It is a hack because the playground does not support displaying multi-part content yet.

Whenever playground gets refactored to properly handle multi-part content, most of this code goes away on the versionToInstance conversion side of things